WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallStepsView.swift

```diff
@@ -0,0 +1,151 @@
+                            .overlay(
+                                Image(uiImage: image)
+                                    .resizable()
+                                    .frame(width: Constants.wooIconSize.width * scale, height: Constants.wooIconSize.height * scale)
+                            )
+                    }
+
+                    Spacer()
+                }
+                .padding(.top, Constants.contentTopMargin)
+
+                // Title and description
+                VStack(alignment: .leading, spacing: Constants.textSpacing) {
+                    Text(Localization.installTitle)
+                        .font(.largeTitle)
+                        .bold()
+                        .foregroundColor(Color(.text))
+
+                    AttributedText(descriptionAttributedString)
+                }
+
+                // Install steps
+                VStack(alignment: .leading, spacing: Constants.stepItemsVerticalSpacing) {
+                    ForEach(JetpackInstallStep.allCases) { step in
+                        HStack(spacing: Constants.stepItemHorizontalSpacing) {
+                            if step == currentStep, step != .done {
+                                ActivityIndicator(isAnimating: .constant(true), style: .medium)
+                            } else if step > currentStep {
+                                Image(uiImage: .checkEmptyCircleImage)
+                                    .resizable()
+                                    .frame(width: Constants.stepImageSize * scale, height: Constants.stepImageSize * scale)
+                            } else {
+                                Image(uiImage: .checkCircleImage)
+                                    .resizable()
+                                    .frame(width: Constants.stepImageSize * scale, height: Constants.stepImageSize * scale)
+                            }
+
+                            Text(step.title)
+                                .font(.body)
+                                .if(step <= currentStep) {
+                                    $0.bold()
+                                }
+                                .foregroundColor(Color(.text))
+                                .opacity(step <= currentStep ? 1 : 0.5)
+                        }
+                    }
+                }
+            }
+            .padding(.horizontal, Constants.contentHorizontalMargin)
+            .scrollVerticallyIfNeeded()
+
+            Spacer()
+
+            // Done Button to dismiss Install Jetpack
+            Button(Localization.doneButton, action: dismissAction)
+                .buttonStyle(PrimaryButtonStyle())
+                .fixedSize(horizontal: false, vertical: true)
+                .padding(Constants.actionButtonMargin)
+        }
+    }
+}
+
+private extension JetpackInstallStepsView {
+    enum Constants {
+        static let contentTopMargin: CGFloat = 69
```

Comment:
This seems to me a constant that we do not use anywhere in the app, perhaps it would be better to align it with others that are closest to our standards?